README.md

```diff
@@ -22,7 +22,8 @@ Javascript implementation (for browser and node.js)
   - [Migrating or Updating versions](#migrating-or-updating-versions)
   - [API](#api)
     - [Constructor(\[url\[, options\]\])](#constructorurl-options)
-    - [options(\[opts\])](#optionsopts)
+    - [getOptions()](#getoptions)
+    - [setOptions(\[options\])](#setoptionsoptions)
```

Comment:
In addition to the above comment — we should keep old one (with deprecation note) and add new ones